actors/builtin/verifreg/verified_registry_actor.go

```diff
@@ -114,16 +120,17 @@ type AddVerifiedClientParams struct {
 }
 
 func (a Actor) AddVerifiedClient(rt vmr.Runtime, params *AddVerifiedClientParams) *adt.EmptyValue {
+	vClientIdAddr, err := builtin.ResolveToIDAddr(rt, params.Address)
+	builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to resolve verified client address %v", params.Address)
```

Comment:
Move after the simple checks, but before the state load

Reply:
Done.